python/pyspark/ml/regression.py

```diff
@@ -68,6 +68,30 @@ class LinearRegression(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPrediction
     Traceback (most recent call last):
         ...
     TypeError: Method setParams forces keyword arguments.
+    >>> import os, tempfile
+    >>> path = tempfile.mkdtemp()
+    >>> lr_path = path + "/lr"
+    >>> lr.save(lr_path)
+    >>> lr2 = LinearRegression.load(lr_path)
+    >>> lr2.getOrDefault(lr2.getParam("maxIter"))
+    5
+    >>> model2 = lr2.fit(df)
```

Comment:
Actually, I don't think you need to fit a second model.  If the goal is to test that all params are the same, that should be done explicitly in a unit test (but it's OK with me not to).